### PR TITLE
Resnest101

### PR DESCRIPTION
**Pull request by akeeei**
1、main.py内加入了读取image_size参数的逻辑
2、renest101模型迁移

### CODE REVIEW

File: cv/classification/main.py [hunk 1]
```diff
@@ -71,8 +71,9 @@ def parse_option():
         "--epochs", type=int, help="batch size for single GPU"
     )
     parser.add_argument("--lr", type=float, help="initial learning rate")
+    parser.add_argument("--image_size", type=int, help="initial image size")
```
Comment (ShawnXuan) [inline cv/classification/main.py:74]: `--image-size`
保持一致的命令行参数命名习惯吧

File: cv/classification/main.py [hunk 2]
```diff
@@ -71,8 +71,9 @@ def parse_option():
...
     parser.add_argument(
-        "--batch-size", type=int, help="batch size for single GPU"
+        "--batch_size", type=int, help="batch size for single GPU"
```
Comment (ShawnXuan) [inline cv/classification/main.py]: `--batch-size`，得遵循之前的命名，改动会影响其他的模型。
`-`是UNIX/Linux命令行参数命名习惯

File: cv/classification/resnest101/infer.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+python3 -m oneflow.distributed.launch \
+        --nproc_per_node $GPU_NUMS \
+        --master_addr 127.0.0.1 \
+        --master_port $PORT \
+        main.py \
+        --cfg configs/resnest_default_settings.yaml \
+        --model_arch $MODEL_ARCH \
+        --image_size $IMAGE_SIZE \
+        --batch_size $BATCH_SIZE \
```
Comment (ShawnXuan) [inline cv/classification/resnest101/infer.sh]: 别忘了修改这里

File: cv/classification/main.py [hunk 4]
```diff
@@ -31,7 +31,7 @@
     AverageMeter,
     TimeMeter
 )
-
+from torch.utils.tensorboard import SummaryWriter
```
Comment (ShawnXuan) [inline cv/classification/main.py]: requirement.txt 中需要增加对`torch`的依赖
有没有必要？

File: cv/classification/main.py (hunk 2 above)
Comment (akeeei) [reply cv/classification/main.py]: 上面46，47行model_arch需要改吗，原来main文件里用的就是
    parser.add_argument(
        "--model_arch",

File: cv/classification/main.py (hunk 4 above)
Comment (akeeei) [reply cv/classification/main.py]: 这里忘记删掉tensorboard内容了，我改一下

**Comment by akeeei** [comment]
image
训练结果
INFO  * Acc@1 81.169 Acc@5 95.477
INFO Accuracy of the network on the 782 test images: 81.2%
INFO Max accuracy: 81.17%
INFO Training time 3:29:19

File: cv/classification/main.py (hunk 2 above)
Comment (ShawnXuan) [reply cv/classification/main.py]: 不用

**Review by ShawnXuan — APPROVED** [review]
